lib/iris/util.py

```diff
@@ -41,6 +41,54 @@
 import iris.exceptions
 
 
+def _cube_cubes_arg(cubes):
+    """
+    Function for ensuring that we return a Cubelist, irrespective of whether
+    a Cube or a CubeList has been provided.
+
+    Args:
+
+    cubes (:class:`iris.cube.Cube` or :class:`iris.cube.CubeList`)
+
+    Returns:
+
+    :class:`iris.cube.CubeList`
+
+    """
+    # Function for ensuring that we always deal with a cubelists
+    if isinstance(cubes, iris.cube.Cube):
+        cubes = iris.cube.CubeList([cubes])
+    return cubes
+
+
+def invert_coordinate(cubes, coordinate):
+    """
+    In-place operation to reverse the direction of a coordinate in a cube or
```

Comment:
Re. "In-place" ... do you need to be in-place for memory reasons? If not, you could make it more robust (and address @ajdawson's comment) by just indexing the cube, e.g. `new_cube = cube[:, :, ::-1, :]`.